modules/stanford_soe_helper_page/stanford_soe_helper_page.info

```diff
@@ -29,32 +29,32 @@ features[ctools][] = views:views_default:3.0
 features[ds_field_settings][] = node|stanford_page|full
 features[ds_layout_settings][] = node|stanford_page|full
 features[features_api][] = api:2
+features[field_base][] = field_s_banner_accent_color
 features[field_base][] = field_s_banner_caption_horizonta
 features[field_base][] = field_s_banner_caption_vertical
 features[field_base][] = field_s_banner_image
 features[field_base][] = field_s_banner_link
 features[field_base][] = field_s_top_banner_caption_horiz
 features[field_base][] = field_s_top_banner_caption_vert
+features[field_base][] = field_s_top_banner_color
 features[field_group][] = group_s_page_banner|node|stanford_page|form
 features[field_group][] = group_s_page_top_banner|node|stanford_page|form
 features[field_group][] = group_s_postcard_content|node|stanford_page|full
 features[field_group][] = group_s_postcard_image|node|stanford_page|full
 features[field_group][] = group_s_postcard|node|stanford_page|full
 features[field_instance][] = node-stanford_page-field_s_apachesolr_index
+features[field_instance][] = node-stanford_page-field_s_banner_accent_color
 features[field_instance][] = node-stanford_page-field_s_banner_caption_horizonta
 features[field_instance][] = node-stanford_page-field_s_banner_caption_vertical
 features[field_instance][] = node-stanford_page-field_s_banner_image
 features[field_instance][] = node-stanford_page-field_s_banner_link
 features[field_instance][] = node-stanford_page-field_s_top_banner_caption_horiz
 features[field_instance][] = node-stanford_page-field_s_top_banner_caption_vert
+features[field_instance][] = node-stanford_page-field_s_top_banner_color
 features[views_view][] = stanford_page_banner_caption
 features[views_view][] = stanford_page_top_banner
-features_exclude[field_base][field_s_banner_accent_color] = field_s_banner_accent_color
```

Comment:
@sherakama they don't want to remove these `features_exclude` lines, do they?

Reply:
They do if they want to track them. See previous comment.

Reply:
OK, per further discussion and investigation: I was confused how the `features_exclude` lines got in here in the first place. There was WIP on these components that got excluded the last time the Feature was exported. So now they _do_ want to include that work, and it's independent of `stanford_page`.